docs/resources/chain-integration.md

```diff
@@ -42,7 +38,7 @@ To provide a stable experience to users, there is a request rate limit on the pu
 4. **EVM HTTP JSON RPC (Web3 compatible)**
    - https://evm-cronos.crypto.org
 
-## Official token contract addresses for Cronos mainnet Beta
+### Official token contract addresses for Cronos mainnet Beta
```

Comment:
@kether-c Kindly update the link to https://cronoscan.com/ in the cases for Cronos mainnet Beta

For example:
```
https://cronos.crypto.org/explorer/address/0x02DCcaf514C98451320a9365C5b46C61d3246ff3/transactions
```
->
```
https://cronoscan.com/address/0x02DCcaf514C98451320a9365C5b46C61d3246ff3
```

Reply:
Sure, updated and pushed